### PR TITLE
Add activation function

### PR DESCRIPTION
**Pull request by kyscg**
Added Swish activation function, along with tests. I'd love to know if I should make any changes or if it is good to merge

Review pleasee @rthadur ????

### CODE REVIEW

File: tensorflow/python/keras/layers/advanced_activations_test.py [hunk 1]
```diff
@@ -59,7 +59,12 @@ def test_thresholded_relu(self):
   def test_softmax(self):
     testing_utils.layer_test(keras.layers.Softmax,
                              kwargs={'axis': 1},
-                             input_shape=(2, 3, 4))
+                             input_shape=(2, 3, 4)
```
Comment (norman-thomas) [inline tensorflow/python/keras/layers/advanced_activations_test.py]: One closing parenthesis was mistakenly removed.

**Comment by kyscg** [comment]
What's the update on this? @rthadur @nataliaponomareva

**Comment by kyscg** [comment]
Could you assign someone else to this @rthadur ???

**Comment by kyscg** [comment]
Gentle ping to inquire why there is no activity here @rthadur ???

**Comment by tanzhenyu** [comment]
Up to @fchollet, but I think it should live in add-on for now and merge into core if it receives wide attention.

**Comment by pavithrasv** [comment]
Yes I agree, we should have this implementation in add ons.

**Comment by tanzhenyu** [comment]
Please create pull requests through tensorflow addons. We would need to consolidate the API before moving it to tensorflow core.

**Pull request closed without merging**